libevm/SmartVM.cpp

```diff
@@ -93,18 +83,11 @@ namespace
 
 		~JitWorker()
 		{
-			DEV_GUARDED(x_mutex)
-				m_finished = true;
-			m_cv.notify_one();
+			push(JitTask::createStopSentinel());
 			m_worker.join();
```

Comment:
Extended comment: this design is quite bad, but it behaves pretty nice in this case. This queue never holds more than one element as JIT compilation finishes before transaction processing finishes.

It would be nice to add clear() here, but std::queue do not have one...